lxd/daemon_images.go

```diff
@@ -26,6 +26,7 @@ import (
 // Simplestream cache
 type imageStreamCacheEntry struct {
```

Comment:
"stream" in the variable name above refers to "SimpleStreams" which is the name of the JSON indexing format used by some image servers. We didn't pick that name :)